tests/lang/static_basic_001.phpt

```diff
@@ -21,10 +21,10 @@ function manyInits() {
     static $counter=0;
     echo "------------- Call $counter --------------\n";
     static $a, $b=10, $c=20, $d, $e=30;
-    echo "Unitialised      : $a\n";
+    echo "Uninitialized    : $a\n";
     echo "Initialised to 10: $b\n";
     echo "Initialised to 20: $c\n";
-    echo "Unitialised      : $d\n";
+    echo "Uninitialized    : $d\n";
     echo "Initialised to 30: $e\n";
```

Comment:
Initially I didn't fold in the en-US change here because I hadn't changed `Initialised`. I'll roll that in at some point.

https://github.com/php/php-src/blob/24388fd5730257f979955238a1b03c2a6870b48a/Zend/zend_execute_API.c#L1612